torchbox/torchbox.com
# Fix three validation errors

No ticket - when testing something else I spotted a few validation errors on the service page.

### Description of Changes Made
- Remove duplicate style attribute from the 'ring' icon
- Output rich text block in div not p
- Fix bug in cookie message include

### How to Test

View a service page's source code and check with https://validator.w3.org/nu/#textarea that there are no errors. You may get one or two where we have more than one image at the same size in a srcset_image - this is due to the way the cropping of images works without upscaling and can be ignored for now.

### Screenshots

<details>
  <summary>Expand to see more</summary>

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
